ecl/hthor/hthor.ipp

```diff
@@ -374,10 +374,18 @@ class CHThorIndexWriteActivity : public CHThorActivityBase
     Owned<IFile> file;
     bool incomplete;
     offset_t sizeLimit;
+    unsigned __int64 duplicateKeyCount;
```

Comment:
minor style: initializer on the member preferred for new code.  (Makes it easier to code review and ensure the constructor always initialises)